envi/archs/amd64/disasm.py

```diff
@@ -544,8 +612,13 @@ def ameth_v(self, bytes, offset, tsize, prefixes, operflags):
 
     def ameth_z(self, bytes, offset, tsize, prefixes, operflags):
         osize, oper = e_i386.i386Disasm.ameth_z(self, bytes, offset, tsize, prefixes, operflags)
-        if not (prefixes & PREFIX_VEX_L):
+
+        if not (prefixes & PREFIX_VEX_L) or operflags & OP_NOVEXL:
```

Comment:
There's a smattering of instructions like psrld, broadcastss, psllw, and others that even though VEX.L is set, for at least one of the operands, the prefix isn't applied and the register is always like an xmm* register, while the other registers are free floating.

Modifying the OPTYPE  size doesn't quite work in those cases because we have to apply the metareg modifiers like RMETA_LOW128.